src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,49 +1,72 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  todo: Todo;
+  onClose: () => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, onClose }) => {
+  const [user, setUser] = useState<User>();
+  const [isLoading, setIsLoading] = useState(true);
+
+  const { id, title, userId } = todo;
+
+  useEffect(() => {
+    getUser(userId).then(resUser => {
+      setUser(resUser);
+      setIsLoading(false);
+    });
+  }, []);
+
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
+      {isLoading
+        ? <Loader />
+        : (
+          <div className="modal-card">
+            <header className="modal-card-head">
+              <div
+                className="modal-card-title has-text-weight-medium"
+                data-cy="modal-header"
+              >
+                {`Todo #${id}`}
+              </div>
 
-      {true ? (
-        <Loader />
-      ) : (
-        <div className="modal-card">
-          <header className="modal-card-head">
-            <div
-              className="modal-card-title has-text-weight-medium"
-              data-cy="modal-header"
-            >
-              Todo #2
-            </div>
+              {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+              <button
+                type="button"
+                className="delete"
+                data-cy="modal-close"
+                onClick={onClose}
+              />
+            </header>
+
+            <div className="modal-card-body">
+              <p className="block" data-cy="modal-title">
+                {title}
+              </p>
+
+              <p className="block" data-cy="modal-user">
+                {
+                  todo.completed
+                    ? <strong className="has-text-success">Done</strong>
+                    : <strong className="has-text-danger">Planned</strong>
```

Comment:
use classnames library for setting needed class instead of rendering separate `strong` elements

Reply:
you can use current condition for the content inside of this tag